zebra-state/src/request.rs

```diff
@@ -318,7 +318,7 @@ impl ContextuallyValidBlock {
     /// including UTXOs created by earlier transactions in this block.
     ///
     /// Note: a [`ContextuallyValidBlock`] isn't actually contextually valid until
-    /// `Chain::update_chain_state_with` returns success.
+    /// `Chain::update_chain_tip_with` returns success.
```

Comment:
Optional: If we link this name, rustdoc will check it is correct for us.

We might need to do an import (with a clippy allow) or provide a full path in the doc to make the docs compile.

```suggestion
    /// [`Chain::update_chain_tip_with()`] returns success.
```